packages/picker/test/index.ts

```diff
@@ -286,6 +286,13 @@ export function runPickerTests(): void {
             expect(el.invalid).to.be.true;
             await expect(el).to.be.accessible();
         });
+        it('renders loading accessibly', async () => {
+            el.loading = true;
+            await elementUpdated(el);
+
+            expect(el.loading).to.be.true;
+            await expect(el).to.be.accessible();
+        });
```

Comment:
I copied the pattern from the invalid property, but I feel like some more tests should be added. Are there any guidelines about writing tests?

Reply:
Two things we'll want to test for:
- is the listbox disabled when `loading`: load, apply loading, try to click, see that `open === false` and the Menu Items cannot be clicked with `sendMouse`
- is the final "loading" label applied to the accessibility tree: load, apply loading, get an accessibility snapshot, ensure there is a node with the expected label

Reply:
Thanks! One additional question about testing: is there any command to run only the tests from one file? I tried some combinations, but failed 😅

Reply:
`yarn test:watch:focus picker`